File: apps/craft-go/models/record.go
package models

// Record represents a user's record.
type Record struct {
	UID                  string    `json:"uid"`
	FirstName            string    `json:"firstName"`
	LastName             string    `json:"lastName"`
	Address              Address   `json:"address"`
	Phone                Phone     `json:"phone"`
	Salary               []Company `json:"salary"`
	TotalHouseholdIncome float64   `json:"totalHouseholdIncome"`
}

File: apps/craft-web/src/app/projects/table/models/record.ts
import { Company } from './company';
import { Phone } from './phone';


export interface Record {
    UID: string;
    name: string;
    avatar: any;
    flicker: any;
    firstName: string;
    lastName: string;
    address: {
      street: string;
      city: string;
      state: string;
      zipcode: string;
    };
    city: string;
    state: string;
    zip: string;
    phone: Phone;
    salary: Company[];
    totalHouseholdIncome: number;
  }